src/k8s/pkg/k8s/setup/certificates.go

```diff
@@ -7,15 +7,20 @@ import (
 )
 
 // InitCertificates sets up the CAs and the necessary server certificates that is used by Kubernetes.
-func InitCertificates() (*cert.CertificateManager, error) {
+// An initial CertificateAuthority can be provided. If not, a self-signed one will be generated.
```

Comment:
minor fix, godoc should start with the function name

Reply:
Not sure if I miss anything, but it does this already, right? It's a multiline comment.